### PR TITLE
Cleanup database between integration tests.

### PR DESCRIPTION
**Pull request by HMCTSDeveloper**
DMP-514

   Implemented clear database before every integration test.

- [X ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-514

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by chrisbellingham-hmcts** [comment]
Hey @HMCTSDeveloper, do we know how this affects test execution time versus what we have today?

**Comment by HMCTSDeveloper** [comment]
> Hey @HMCTSDeveloper, do we know how this affects test execution time versus what we have today?

**Comment by HMCTSDeveloper** [comment]
> Hey @HMCTSDeveloper, do we know how this affects test execution time versus what we have today?

It went roughly from 13~ to 17~ seconds.

File: src/integrationTest/java/uk/gov/hmcts/darts/common/util/ClearDatabase.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation drops all objects in the database and the recreates it using flyway.
+ */
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(ClearDatabaseExtension.class)
+@TestPropertySource(properties = "spring.flyway.clean-disabled=false")
```
[the remark this answers is not in the document]
Comment (HMCTSDeveloper) [reply src/integrationTest/java/uk/gov/hmcts/darts/common/util/ClearDatabase.java:17]: The property is saying to allow database clean. It's a weirdly phrased property.

**Review by chrisbellingham-hmcts — APPROVED** [review]

**Pull request merged**